benchmarks/float/cordic.out

```diff
@@ -0,0 +1 @@
+0.86369602123419631
```

Comment:
Just confirming with a reference implementation:

```py
>>> import math
>>> math.sin(1.0472)
0.8660266281835431
```

Accurate to two decimal places! Not too shabby.